app/src/main/java/com/example/photogallery/presentation/VisibleFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.photogallery.presentation
+
+import android.app.Activity
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.util.Log
+import androidx.fragment.app.Fragment
+import com.example.photogallery.data.PollWorker
+
+abstract class VisibleFragment : Fragment() {
```

Comment:
Vlad had a comment about this fragment's name on Alexandra's pull request and it applies here too:
https://github.com/alexandragnj/PhotoGalery/pull/5#discussion_r908093237

Reply:
Done! :)